server/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/jak103/uno
+module github.com/jak103/usu-uno
```

Comment:
Unless something has changed since I last talked to @jak103 on Thursday, I think we want to leave the module named `jak103/uno`. Is this breaking something if left as `jak103/uno`?

Reply:
I found that this will run locally if I replace `jak103/usu-uno` in each of `go.mod`, `deckUtils.go`, and `deckUtils_test.go`.

Reply:
I had it as just uno, and the coverage action failed during testing. Failure started when the implementation of the card was changed from the UnoCard struct within DeckUtils.go, to the Card struct in the model package located in uno/server/model. It doesn't work either way because when my tests go to run, they are unable to find the Card struct because it is looking in "jak103/uno" when it needs to look in "jak103/uno/server/". I submitted a different PR trying address this, https://github.com/jak103/uno/pull/109.
I don't fully understand all of the moving pieces but my tests ran great prior to the changes made to the Card implementation. Thanks for the help!

Reply:
I think the failing is caused because of a failing test -- the current card implementation is currently not dealing any cards, so the unit test checking for the winner is failing, which would cause tests to fail. We shouldn't have to look in `jak103/uno/server` -- that's the base folder of the module, so only the folder structure past `server` matters.

Reply:
I can hop in discord and look at this with you, if you'd like.

Reply:
Yeah that would be helpful for me!